### PR TITLE
Allow use of prefixes and consumer groups

### PR DESCRIPTION
**Pull request by ahamidi**

### CODE REVIEW

**Comment by kvlou** [comment]
One comment but it looks good! :+1:

**Comment by ahamidi** [comment]
@kvlou What's the comment? (Also apologies for the reviewers...meant to add as assignees)

File: main.go [hunk 1]
```diff
@@ -227,3 +232,31 @@ func (ac *AppConfig) brokerAddresses() []string {
...
 }
+
+// Prepends prefix to topic if provided
+func (ac *AppConfig) topic() string {
+	var topic string
+
+	if ac.Kafka.Topic != "" {
+		topic = ac.Kafka.Topic
+	} else {
+		topic = "messages"
```
Comment (halorgium) [inline main.go]: this has a default already.

Comment (ahamidi) [inline main.go]: oops

**Pull request merged**